di-ipv-core-stub/src/main/java/uk/gov/di/ipv/stub/core/config/uatuser/IdentityMapper.java

```diff
@@ -114,6 +142,11 @@ public List<QuestionAndAnswer> mapToQuestionAnswers(
     }
 
     public Identity mapFormToIdentity(Identity identityOnRecord, QueryParamsMap userData) {
+        List<UKAddress> addresses = new ArrayList<>();
+
+        LocalDate vf = getLocalDate(userData, "validFrom-year", "validFrom-month", "validFrom-day");
+
+        LocalDate vu = getLocalDate(userData, "validFrom-year", "validFrom-month", "validFrom-day");
 
         UKAddress ukAddress =
```

Comment:
rename ukAddress to a more specific name
e.g. `primaryAddress` / `firstAddress` / `currentAddress` etc